files/besu/README.md

```diff
@@ -129,19 +129,28 @@ There is an additional erc20 token example that you can also test with: executin
 This can be verified from the `data` field of the `logs` which is `1`.
 
 ### iii. Smart Contracts & DApps <a name="poa-network-dapps"></a>
-- Once you have a network up and running from above, install [metamask](https://metamask.io/) as an extension in your browser
-- Once you have setup your own private account, select 'My Accounts' by clicking on the avatar pic and then 'Import Account' and enter the valid private_key
-- You can now deploy contracts and connect DApps to the network. 
-![Image dapp](./static/qs-dapp.png)
-
-As seen in the architecture overview diagram you can extend the network with monitoring, logging, smart contracts, DApps and so on
-
 As an example we've included the Truffle Pet-Shop Dapp in the `dapps` folder and here is a [video tutorial](https://www.youtube.com/watch?v=_3E9FRJldj8) you 
 can follow of deployment to the network and using it. Please import the private key `0xc87509a1c067bbde78beb793e6fa76530b6382a4c0241e5e4a9ec0a0f44dc0d3` to
 Metmask **before** proceeding to build and run the DApp with `run-dapp.sh`. Behind the scenes, this has used a smart contract that is compiled and then 
 deployed (via a migration) to our test network. The source code for the smart contract and the DApp can be found in the folder `dapps/pet-shop`
 
+
+| ⚠️ **WARNING**:  This is a test account only and the private and public keys are publicly visible. **Using test accounts on Ethereum mainnet and production networks can lead to loss of funds and identity fraud.** In this documentation, we only provide test accounts for ease of testing and learning purposes;never use them for other purposes. **Always secure your Ethereum mainnet and any production account properly.** See for instance [MyCrypto "Protecting Yourself and Your Funds" guide](https://support.mycrypto.com/staying-safe/protecting-yourself-and-your-funds).  |
```

Comment:
Not sure about the format, could we have a BIG warning as you did but then have the text on multiple lines, perhaps bullet list because here it's not easy to read in one block.

Reply:
I'll try that now

Reply:
Got em on seperate lines where the bold and normal are at least showing for emphasis but not quite bullets yet. I'll do a release of this for now so it's at least there and visible